### PR TITLE
FEAT: show tags in crawler view of tags page for static site

### PR DESCRIPTION
**Pull request by mrfinch**
This adds tags layout in crawler view for a static site. Attaching screenshot for the same below.
This is a part of https://meta.discourse.org/t/improving-discourse-static-html-archive/112497

Screenshot from 2019-04-24 21-58-00

### CODE REVIEW

**Comment by gschlager** [comment]
For anyone else looking at this PR: The logic in the `TagsController` is completely the same. The diff only looks that bad because there was some shuffling around to introduce the `@tags` and `@extras` variables.

File: app/views/tags/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+          <%= render "tag", tag: tag %>
+        <% end %>
+      </div>
+      <div class="clearfix"></div>
+    <% end %>
+  <% end %>
+
+  <% if @tags && !@tags.empty? %>
+    <div class="tag-list">
+      <h3>Others</h3>
```
[the remark this answers is not in the document]
Comment (mrfinch) [reply app/views/tags/index.html.erb]: ahh again. I have made the fix @gschlager, thanks

[the remark this answers is not in the document]
Comment (mrfinch) [reply app/views/tags/index.html.erb]: @gschlager ping :bowing_man:

File: app/views/tags/_tag.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="tag-box">
+  <a href="/tags/<%= tag[:id] %>" class="discourse-tag simple"><%= tag[:text] %></a>
```
Comment (ZogStriP) [inline app/views/tags/_tag.html.erb]: That link won't work when using subfolder instance.

Comment (mrfinch) [reply app/views/tags/_tag.html.erb]: I have tried to fix it, let me know if it works :slightly_smiling_face:  @ZogStriP

File: app/views/tags/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+        <% end %>
+        <% tag_group[:tags].each do |tag| %>
+          <%= render "tag", tag: tag %>
+        <% end %>
+      </div>
+      <div class="clearfix"></div>
+    <% end %>
+  <% end %>
+
+  <% if @tags && !@tags.empty? %>
```
Comment (ZogStriP) [inline app/views/tags/index.html.erb]: This can be shortened to

```suggestion
  <% if @tags.present? %>
```

Comment (mrfinch) [reply app/views/tags/index.html.erb]: I have made the change @ZogStriP, sorry for the delay :bowing_man:

Comment (mrfinch) [reply app/views/tags/index.html.erb]: @ZogStriP ping

**Comment by SamSaffron** [comment]
This looks pretty safe to me, merging this in for now and will keep an eye on it. `?w=1` is your github friend here. 

Thanks!

**Pull request merged**